template.yml

```
@@ -64,3 +64,16 @@ Resources:
             Properties:
               Path: /v3/technical-records/{systemNumber}/{createdTimestamp}
               Method: patch
+    UpdateLambdaFunction:
+      Type: 'AWS::Serverless::Function'
+      Properties:
+        CodeUri: src/handler/
+        Handler: update.handler
+        Runtime: nodejs18.x
+        Timeout: 20
+        Events:
+          UpdateLambda:
+            Type: Api
+            Properties:
+              Path: /v3/technical-records/update/{systemNumber}/{createdTimestamp}
```

```suggestion
              Path: /v3/technical-records/{systemNumber}/{createdTimestamp}
```
Why doesn't this live under this route? You can have multiple verbs on the same route. And the `POST`/`PUT` verb on this route is not curretnly utilies